docs/tutorial/part-one/index.md

```diff
@@ -365,6 +365,8 @@ Gatsby.js is a _modern site generator_, which means there are no servers to set
 
 Try using [Surge](http://surge.sh/) for deploying your first Gatsby website. Surge is one of many "static site hosts" which makes it possible to deploy Gatsby sites.
 
+> Gatsby Cloud is another deployment option, built by the team behind Gatsby.js. You can find instructions for deploying to Gatsby Cloud [below](https://www.gatsbyjs.org/tutorial/part-one/#alternative-deploying-to-gatsby-cloud).
```

Comment:
Have you read the Gatsby style guide?  We are always Gatsby, never 'Gatsby.js'

Reply:
Thanks for pointing this out! I hadn't seen that part of the style guide. The tutorial actually uses 'Gatsby.js' just a few lines above, so I suppose that should be updated too.

![Screenshot from Gatsby tutorial](https://user-images.githubusercontent.com/4264038/83185403-ee62b000-a0f8-11ea-843f-13165e8ea18f.png)